### PR TITLE
Add django-massmailer integration

### PR DESCRIPTION
**Pull request by seirl**
The style of some parts is broken but that's because your CSS is weird.

### CODE REVIEW

File: requirements/production.txt [hunk 1]
```diff
@@ -4,6 +4,7 @@ django-allauth==0.38.0
 django-bootstrap4
 django-celery-beat==1.1.*
 django-js-reverse==0.8.2
+django-massmailer
```
Comment (RaitoBezarius) [inline requirements/production.txt]: Can you pin a version?

File: mangaki/mangaki/settings.py [hunk 2]
```diff
@@ -69,7 +69,11 @@
     'bootstrap4',
     'django_js_reverse',
     'rest_framework',
-    'django_celery_beat'
+    'django_celery_beat',
+    'massmailer',
+    'crispy_forms',
+    'django_bootstrap_breadcrumbs',
+    'reversion',
```
Comment (RaitoBezarius) [inline mangaki/mangaki/settings.py]: What is reversion?

File: requirements/production.txt (hunk 1 above)
Comment (seirl) [reply requirements/production.txt]: Considering I'm still unbreaking stuff on it, I don't really advise it. You're the first early adopters!

File: mangaki/mangaki/settings.py (hunk 2 above)
Comment (seirl) [reply mangaki/mangaki/settings.py]: A dependency of massmailer that is used to store past versions of the templates and queries: https://github.com/etianen/django-reversion

**Comment by seirl** [comment]
> Sounds good, what about the "weird CSS" stuff? Can you elaborate or be more precise? Can we fix it on our own?

It's basically because your CSS is too broad, so it changes stuff that don't have class attributes, which breaks some parts of the design.

**Comment by seirl** [comment]
(Please note the draft status of this, I'm still making some changes and looking for early feedback, don't merge this)

**Comment by RaitoBezarius** [comment]
Yes, I understood, I was taking an early look to it as I was curious. Thank
you for this !

**Comment by RaitoBezarius** [comment]
Closed in favor of #765 which will use listmonk for now.
Deep-integration with Mangaki is yet to be done, but unnecessary for now.

**Pull request closed without merging**